nvcodec/samples/pom.xml

```diff
@@ -0,0 +1,30 @@
+<project>
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.bytedeco.nvcodec</groupId>
+    <artifactId>samplenvcodec</artifactId>
+    <version>1.5.6-SNAPSHOT</version>
+    <dependencies>
+        <dependency>
+            <groupId>org.bytedeco</groupId>
+            <artifactId>nvcodec-platform</artifactId>
+            <version>11.0.10-1.5.6-SNAPSHOT</version>
+        </dependency>
+
+        <!-- Additional dependencies to use bundled CUDA -->
+        <dependency>
+            <groupId>org.bytedeco</groupId>
+            <artifactId>cuda-platform-redist</artifactId>
+            <version>11.2-8.1-1.5.6-SNAPSHOT</version>
+        </dependency>
+    </dependencies>
+    <build>
+        <resources>
+            <resource>
+                <directory>src/main/java</directory>
+            </resource>
+            <resource>
+                <directory>src/main/resources</directory>
+            </resource>
+        </resources>
+    </build>
```

Comment:
I don't think we need to specify the directories here since those are the defaults for Maven, right?

Reply:
Right, i will remove. xD